ygot/struct_validation_map.go

```diff
@@ -468,10 +468,9 @@ func copyInterfaceField(dstField, srcField reflect.Value) error {
 
 // copyMapField copies srcField into dstField. Both srcField and dstField are
 // reflect.Value structs which contain a map value. If both srcField and dstField
-// are populated, and have non-overlapping keys, they are merged. If a key exists
-// in both the srcField and dstField map, an error is returned.
-// TODO(robjs): Implement merging maps where there are key values taht overlap.
-//  See https://github.com/openconfig/ygot/issues/74.
+// are populated, and have non-overlapping keys, they are merged. If the same
+// key is populated in srcField and dstField, their contents are merged unless
+// there are overlapping fields populated within the map values.
```

Comment:
suggestion: ...dstField, their contents are merged if they do not overlap, or an error is returned if they overlap.? something like that.

Reply:
Done, thanks.